os_net_config/impl_nmstate.py

```diff
@@ -45,7 +45,6 @@
 from os_net_config import objects
 from os_net_config import utils
 
-
```

Comment:
It's not unusual to have two white space lines after includes, but I'm not attached to either way.